### ISSUE TITLE
Maps dont show Pokestops

### ISSUE THREAD

**Issue by Miiwii** [finndev/PokeBuddy#693]
He guyz
my Problem is when i start the Bot there are no Pokestops on my map anymore.
I choose a city and i see that the coords are correct.
I reinstaled the bot 3 times now and also made my antivirus off but nothing changened.
Can someone help me with that problm pls ?

**Comment by footekruk** [comment]
Same problem here. Seems to do it randomly, but pretty constant at the moment.

**Comment by psblackz** [comment]
just use 1 bot

**Comment by curtiola** [comment]
Was the Antivirus off when downloading the zip? It may have deleted something, I doubt it would without notifying you though, sometimes you just have to restart your pc

**Comment by Miiwii** [comment]
Yes it was off and i only used 1 bot 
I m playing on w10 64bit

**Comment by curtiola** [comment]
Idk for sure, some people had it work when they lowered their walk speed below 25, the solutions isn't always the same lol

**Comment by Miiwii** [comment]
Yesterday i made the update for incubator and then the problem came 3 hours later

**Comment by curtiola** [comment]
Yesterday Niantic updated pogo too, added speed limits and stuff, if you go over 65 with the app open it soft bans you and stuff like that (road safety and whatnot) try lowering walk speed to 25?

**Comment by inphiniti** [comment]
i tryed just about everything having the same problem

**Comment by footekruk** [comment]
Had it running all day at 100 today. Seems to just be a recent change in it. Maybe update finally started affecting it.

**Comment by Miiwii** [comment]
the bot worked after the niantic update with the settings and now it doesnt work oO

**Comment by curtiola** [comment]
I could be wrong, I'm going off the word of others, but there have been others with this problem, sometimes they redownload and it works and 3 of them said it worked after lowering walk speed to 25, and if you can't see items or level either, a pc restart is required

**Comment by curtiola** [comment]
Did it just start very recently? It could be pogo server related

**Comment by footekruk** [comment]
Had a couple random issue earlier in the day but they would fix themselves rather quickly. All poke stops stopped showing up like 45-60 min ago.

**Comment by Miiwii** [comment]
no it doesnt show anything on my map (no stops and no pokemon to catch)
i restarted my pc and also set speed to 10

**Comment by EpicKeith** [comment]
My speed is only 4 KM/h - worked fine until 20 minutes ago, having the same issue at the moment.
On my phone the stops still show so I assume it's server based.

**Comment by HenghaoZhou** [comment]
For me. It worked 20 mins and then all the pokestops disappeared and then just let it run another 20min it will work well again. So 20 mins by 20mins.......

**Comment by EpicKeith** [comment]
Works on my end again, probably a server issue?

**Comment by curtiola** [comment]
Is it possible your inventory is completely full? I assume not but that has caused some issues in older versions, and someone in another thread just solved it by evolving/transferring some pidgeys and stuff on his phone lol #694

**Comment by curtiola** [comment]
It was probably server related if yours and his started working at almost the exact same time

**Comment by Miiwii** [comment]
yes its working now :)

**Comment by Miiwii** [comment]
thanks guyz

**Comment by curtiola** [comment]
Np, mind closing the issue? The devs have a lot to filter through haha

**Comment by EpicKeith** [comment]
And it's back again.

**Comment by curtiola** [comment]
The servers are under a lot more pressure than Niantic expected lately, they shut down all the bots and maps and such before releasing to Brazil and south Asia for that reason, and now the bots and scrapers are back so I expect things to be choppy for a while, I just hope they build more servers instead of trying to shut down bots again haha

**Comment by curtiola** [comment]
Lol if anything I'd just plead for them to upgrade servers instead of wasting their time and money on another API change or some other anti bot method that would be cracked in 3 days again xD

**Comment by JCortexs** [comment]
its working for me!

**Comment by AlbRk** [comment]
Just crashed

**Comment by thegodwarrior** [comment]
Still no pokestops or gyms..

**Comment by footekruk** [comment]
I was just talking about how good the servers have been... tsk tsk tsk

**Comment by Thorben24** [comment]
@Miiwii @thegodwarrior @footekruk 

Had the same problem some hours ago.

I fixed it by changing my ip adress. You can usually do this in few seconds in your router settings or by restarting your router (power off, power on).

**Comment by LeeTheOrc** [comment]
same problem here too

**Comment by thegodwarrior** [comment]
@Thorben24 thanks man, it worked after i restarted my modem

**Comment by thegodwarrior** [comment]
The same problem happens again after botting awhile..

**Comment by Rekario** [comment]
Confirm IP-Ban. Running QuickSpin and PokeBuddy throught proxy and all is working without problems. Changing proxys like my underwear x)

**Comment by xSTRYKAx** [comment]
It's definitely an ip ban guys, I am now using a VPN (with my walk speed lowered to 25) and it's working fine again!

**Comment by Miiwii** [comment]
Which vpn you use Stryka?

**Comment by xSTRYKAx** [comment]
It's called Surfeasy, available from cnet!

**Comment by Miiwii** [comment]
Thanks bro !

**Comment by xSTRYKAx** [comment]
Yw, just for info, i got ip banned again so i switched my location via the VPN, turned incubator off (as i had no problems before turning it on) and lowered my walk speed to 20, so far so good and i'm gaining just as much xp at the lower speed!

**Comment by Miiwii** [comment]
yeah i tried it with teleport and it was working for 45 minutes before i got an ip ban without vpn :D

**Comment by DUSKvsDAWN** [comment]
same, not banned atm for as far as I know. but even IF I'm banned normally it does show all Pokestops.
